src/store/files.js

```diff
@@ -297,6 +297,11 @@ export const useFilesStore = function(...args) {
 
 				if (filter) {
 					for (const [key, value] of Object.entries(filter)) {
+						if (key === 'nodeId') {
+							params.set('nodeIds[]', value)
+							continue
+						}
+
```

Comment:
I think that will be best to change the point that send `nodeId` to instead of, send the `nodeIds`